aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIOProperties.java

```diff
@@ -498,6 +506,8 @@ public S3FileIOProperties() {
     this.s3RetryNumRetries = S3_RETRY_NUM_RETRIES_DEFAULT;
     this.s3RetryMinWaitMs = S3_RETRY_MIN_WAIT_MS_DEFAULT;
     this.s3RetryMaxWaitMs = S3_RETRY_MAX_WAIT_MS_DEFAULT;
+    this.isTreatS3DirectoryBucketListPrefixAsDirectoryEnabled =
```

Comment:
the name of the variable and the constant does not match, can we just call this `s3DirectoryBucketListPrefixAsDirectory`?